client/hospital.lua

```diff
@@ -64,14 +64,15 @@ local function putPlayerInBed(hospitalName, bedIndex, isRevive, skipOpenCheck)
     exports.qbx_medical:DisableDamageEffects()
     exports.qbx_medical:disableRespawn()
     CanLeaveBed = false
+    local numDoctors = lib.callback.await('qbx_ambulancejob:server:getNumDoctors')
     setBedCam()
     CreateThread(function()
         Wait(5)
         if isRevive then
             exports.qbx_core:Notify(Lang:t('success.being_helped'), 'success')
             Wait(config.aiHealTimer * 1000)
-            TriggerEvent("hospital:client:Revive")
-        else
+            TriggerEvent("qbx_medical:client:playerRevived")
```

Comment:
Thinking this should be an export. In general I don't think resource A should trigger events owned by resource B. API should be provided via exports.